pkg/minikube/bootstrapper/bsutil/ktmpl/v1alpha3.go

```diff
@@ -57,6 +57,7 @@ etcd:
   local:
     dataDir: {{.EtcdDataDir}}
 controllerManagerExtraArgs:
+  allocate-node-cidrs: "true"
```

Comment:
I am curious, other versions v1beta1 and 1vbeta2 need no addition to yam ?
I think this yaml file is only used for very old kuberentes versions...

Reply:
For other versions template it comes from the `versionSpecificOpts` options in version.go.